hudi-client/hudi-spark-client/src/main/java/org/apache/hudi/index/bloom/BucketizedBloomCheckPartitioner.java

```diff
@@ -90,7 +90,8 @@ public BucketizedBloomCheckPartitioner(int targetPartitions, Map<HoodieFileGroup
     // of buckets and assigns buckets in the same order as file groups. If we were to simply round robin, then buckets
     // for a file group is more or less guaranteed to be placed on different partitions all the time.
     int minBucketsPerPartition = Math.max((int) Math.floor((1.0 * totalBuckets) / partitions), 1);
-    LOG.info(String.format("TotalBuckets %d, min_buckets/partition %d", totalBuckets, minBucketsPerPartition));
+    LOG.info(String.format("TotalBuckets %d, min_buckets/partition %d, partitions %d", totalBuckets, minBucketsPerPartition,
```

Comment:
change this to 
Log.info("TotalBuckets {}, min_buckets/partition {}, partitions {}", totalBuckets, minBucketsPerPartition, partitions);